migrations/default/2023_03_24_125122_add_id_to_related.php

```diff
@@ -27,8 +27,10 @@ public function up(): void
 
     public function down(): void
     {
-        Schema::table(config('twill.related_table', 'twill_related'), function (Blueprint $table) {
-            $table->dropColumn('id');
-        });
+        if (! Schema::hasColumn(config('twill.related_table', 'twill_related'), 'id')) {
```

Comment:
It's the other way around

```suggestion
        if (Schema::hasColumn(config('twill.related_table', 'twill_related'), 'id')) {
```